src/js/controllers/index.js

```diff
@@ -274,6 +278,7 @@ angular.module('copayApp.controllers').controller('indexController', function($r
         self.walletScanStatus = walletStatus.wallet.scanStatus;
         self.copayers = walletStatus.wallet.copayers;
         self.preferences = walletStatus.preferences;
+        $rootScope.$emit('Local/BalanceUpdated', walletStatus.balance);
```

Comment:
As I outlined in PR description, colored addon subscribes to this event: https://github.com/troggy/copay-colored-coins-plugin/blob/master/js/controllers/assets.js#L9